src/modeler/AutoRest.Swagger/Model/ServiceDefinition.cs

```diff
@@ -83,6 +83,7 @@ public ServiceDefinition()
         [Rule(typeof(UniqueResourcePaths))]
         [Rule(typeof(ListOperationNamingWarning))]
         [Rule(typeof(ListByOperationsValidation))]
+        [Rule(typeof(CollectionObjectPropertiesNamingValidation))]
```

Comment:
Since we have 2 "paths" sections, one for the regular paths and one for our "x-ms-paths" both of which could theoretically violate the rule (as indicated in the test jsons)